modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/ScheduleField.tsx

```diff
@@ -50,6 +50,18 @@ export default function ScheduleField({
 		[checked, customValidation]
 	);
 
+	useEffect(() => {
+		const handleSubmit = () => {
+			handleError(value);
+		};
+
+		Liferay.on('closeModalSchedulePublication', handleSubmit);
+
+		return () => {
+			Liferay.detach('closeModalSchedulePublication', handleSubmit);
+		};
+	}, [handleError, value]);
```

Comment:
Hi @larissacribeiro, the event listener will be responsible for displaying errors in the fields using submitRef, right? The JSP will be responsible for closing the modal in case of review or expiration errors, right?

So why is this necessary?